src/content/pages/faq.mdx

```diff
@@ -5,11 +5,37 @@ subtitle: Frequently Asked Questions about EuroPython
 
 # FAQ
 
+## **Milestones**
+<ul className="milestone-done">
+<li><b>2024-01-27</b>: [Call for Contributors](/https://blog.europython.eu/seize-the-opportunity-europython-2024-call-for-contributors-extended/) Sign-up Starts </li>
+<li><b>2024-02-12</b>: [Call for Contributors](/https://blog.europython.eu/seize-the-opportunity-europython-2024-call-for-contributors-extended/) Sign-up Closes </li>
+<li><b>2024-02-14</b>: [Speaker Mentorship Programme](/mentorship) Sign-up for Mentees Starts </li>
+<li><b>2024-02-14</b>: [Call for Proposals (CFP)](/cfp) Starts </li>
+<li><b>2024-02-27</b>: [Speaker Mentorship Programme](/mentorship) Sign-up for Mentes Closes </li>
+<li><b>2024-03-08</b>: [Financial Aid Programme](/finaid) Starts </li>
+<li><b>2024-02-26</b>: [Speaker Mentorship Programme Ask Me Anything Event](/mentorship#ask-me-anything-about-cfp-workshop---tuesday-14-march-2023-1800-cet)</li>
```

Comment:
I think the year on the URL is incorrect, but I'm not sure what's the equivalent.

Reply:
Just added the general mentorship page